platform/viewer/pluginConfig.json

```diff
@@ -27,12 +27,17 @@
     {
       "packageName": "@ohif/extension-tmtv",
       "version": "3.0.0"
+
     }
   ],
   "modes": [
     {
       "packageName": "@ohif/mode-longitudinal",
       "version": "3.0.0"
+    },
+    {
```

Comment:
I will undo this change before commit, but it is required to test the HP changes.